invenio_rdm_records/marshmallow/json.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""JSON Schemas."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_records_rest.schemas import Nested, StrictKeysMixin
+from invenio_records_rest.schemas.fields import DateString, \
+    PersistentIdentifier, SanitizedUnicode
+from marshmallow import fields, validate
+
+
+class AccessSchemaV1(StrictKeysMixin):
+    """Access schema."""
+
+    metadata_restricted = fields.Bool(required=True)
+    files_restricted = fields.Bool(required=True)
```

Comment:
Made an issue to cover implementation / discussion on this: #20 . Maybe we can put a `# TODO:` on those metadata fields we haven't gone through yet.

Reply:
:+1: